spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/TableHelper.java

```diff
@@ -0,0 +1,20 @@
+package uk.ac.bbsrc.tgac.miso.spring.ajax;
+
+public class TableHelper {
+
+  // hyperLinkify(String, String) returns the hyperlink-ed (in HTML) version of
+  // text given a path. 
```

Comment:
Good question. Likely, but it looks like it was written with JS styling. I'll fix that.